build.sh

```diff
@@ -364,7 +364,7 @@ fi
 if [ $__CrossBuild == 1 ]; then
     export CROSSCOMPILE=1
     if ! [[ -n "$ROOTFS_DIR" ]]; then
-        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs/$__BuildArch"
+        export ROOTFS_DIR="$__ProjectRoot/cross/rootfs"
     fi
```

Comment:
I decided on removing it there because it would be confusing for a user to set it to a value and being overridden during the script execution.